src/types/index.tsx

```diff
@@ -50,3 +50,17 @@ export type Session = {
   start_date: string;
   year: number;
 };
+
+export type Attendance = {
+  date: string;
+  attendees: number[];
+};
+
+export type ClassInfo = {
+  id: number;
+  name: string;
+  session_id: number;
+  facilitator_id: number;
+  attendance: Attendance[];
+  families: Family[];
```

Comment:
the api won't return everything on the `Family` type - it'll be the [`FamilyListResponse` type](https://github.com/uwblueprint/project-read-frontend/blob/ca6fb7b49684339d0322337eca8a327b374cc807/src/api/FamilyAPI.ts#L21)!